consensus/istanbul/backend/backend.go

```diff
@@ -621,6 +621,17 @@ func (sb *Backend) Verify(proposal istanbul.Proposal) (*istanbulCore.StateProces
 		}
 	}
 
+	if sb.chain.Config().IsGingerbreadP2(block.Number()) {
+		bytesBlock := new(core.BytesBlock).AddBytes(params.MaxTxDataPerBlock)
```

Comment:
nit: The code to check this exists twice, you could factor it out. But it's not necessary for this PR.